src/cell.rs

```diff
@@ -415,15 +448,15 @@ mod test {
     #[test]
     fn angles() {
         let mut cell = UnitCell::new([2.0, 3.0, 4.0]);
-        assert_eq!(cell.angles(), [90.0, 90.0, 90.0]);
+        ::assert_vector3d_eq(&cell.angles(), &[90.0, 90.0, 90.0], 1e-6);
```

Comment:
I find this easier to understand when trying to find the definition of `assert_vector3d_eq`

```suggestion
        crate::assert_vector3d_eq(&cell.angles(), &[90.0, 90.0, 90.0], 1e-6);
```

Reply:
I didn't think much about that at the time of writing this code, but now I see the benefit :=)